src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -100,15 +103,53 @@ private void LogCommonPropertiesAndNewLine(IOperation operation)
                 LogString(", IsImplicit");
             }
 
-
             LogString(")");
 
             // Syntax
-            LogString($" (Syntax: {GetSnippetFromSyntax(operation.Syntax)})");
+            LogString($" (Syntax: {GetKind(operation.Syntax)}, {GetSnippetFromSyntax(operation.Syntax)})");
+
+            // Parent
+            if (operation == _root)
+            {
+                LogString($" (Parent: {operation.Parent?.Kind})");
```

Comment:
:bulb: This change can be implemented alongside the previous one (showing the current syntax kind).